src/preprocessing/util/ite_utilities.cpp

```diff
@@ -1064,13 +1064,13 @@ Node ITESimplifier::transformAtom(TNode atom)
   }
 }
 
-static unsigned numBranches = 0;
-static unsigned numFalseBranches = 0;
-static unsigned itesMade = 0;
+static thread_local unsigned numBranches = 0;
+static thread_local unsigned numFalseBranches = 0;
+static thread_local unsigned itesMade = 0;
```

Comment:
This looks more like it should statistic? members in the ITESimplifier itself?

Reply:
I was trying to alter the code as little as possible to make it work when multithreaded, but happy to make those changes if it's a good idea.

Reply:
This is similar to repCheckInstance below. For this PR it is OK  just to make it thread_local. It would be good to make follow up PR to eliminate these static variables.